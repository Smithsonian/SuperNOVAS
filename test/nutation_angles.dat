
2450545.000000 TEST       S2 O0 A1: -40700.089442  1264.829260

2450545.000000 TEST       S2 O1 A1: -40700.089442  1264.829260

2450545.000000 TEST       S2 O2 A1: -40700.089442  1264.829260

2450545.000000 SUN        S0 O0 A1: -40700.089442  1264.829260

2450545.000000 SUN        S0 O1 A1: -40700.089442  1264.829260

2450545.000000 SUN        S0 O2 A1: -40700.089442  1264.829260

2451545.000000 TEST       S2 O0 A1: 17706.997213 -1682.468119

2451545.000000 TEST       S2 O1 A1: 17706.997213 -1682.468119

2451545.000000 TEST       S2 O2 A1: 17706.997213 -1682.468119

2451545.000000 SUN        S0 O0 A1: 17706.997213 -1682.468119

2451545.000000 SUN        S0 O1 A1: 17706.997213 -1682.468119

2451545.000000 SUN        S0 O2 A1: 17706.997213 -1682.468119

2452545.000000 TEST       S2 O0 A1: -26565.908427  2576.032917

2452545.000000 TEST       S2 O1 A1: -26565.908427  2576.032917

2452545.000000 TEST       S2 O2 A1: -26565.908427  2576.032917

2452545.000000 SUN        S0 O0 A1: -26565.908427  2576.032917

2452545.000000 SUN        S0 O1 A1: -26565.908427  2576.032917

2452545.000000 SUN        S0 O2 A1: -26565.908427  2576.032917
