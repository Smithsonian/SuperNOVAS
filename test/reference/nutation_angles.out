  -10000.000 A0: current    16.415313     4.048559
  -10000.000 A0: future     -4.602182    -8.501597
       0.000 A0: current   -13.931996    -5.769398
       0.000 A0: future      3.271334     8.585415
   10000.000 A0: current    10.602901     6.295105
   10000.000 A0: future     -0.608090    -9.492474
   10000.000 A0: current    10.602901     6.295105
   10000.000 A0: future     -0.608090    -9.492474
   10000.010 A0: current    10.603323     6.294395
   10000.010 A0: future     -0.608779    -9.492433
  -10000.000 A1: current    16.415324     4.048553
  -10000.000 A1: future     -4.602205    -8.501588
       0.000 A1: current   -13.931977    -5.769410
       0.000 A1: future      3.271330     8.585420
   10000.000 A1: current    10.602913     6.295092
   10000.000 A1: future     -0.608099    -9.492480
   10000.000 A1: current    10.602913     6.295092
   10000.000 A1: future     -0.608099    -9.492480
   10000.010 A1: current    10.603335     6.294382
   10000.010 A1: future     -0.608787    -9.492439
